Code/DataStructs/Wrap/testBV.py

```diff
@@ -314,5 +314,33 @@ def test10BulkOps3(self):
             self.assertTrue(feq(sim, sims[i]))
 
 
+    def test11BulkNeighbors(self):
+        nbits = 2048
+        bvs = []
+        for bvi in range(1000):
+            bv = DataStructs.ExplicitBitVect(nbits)
+            for j in range(nbits):
+                x = random.randrange(0, nbits)
+                bv.SetBit(x)
+            bvs.append(bv)
+        qs = bvs[:10]
+        db = bvs[10:]
+        tgts = []
+        for q in qs:
+            sims = DataStructs.BulkTanimotoSimilarity(q,db)
```

Comment:
It seems like there are a few metrics not being tested.  I'm not sure if the is necessary, but could be done.

Reply:
sure... can do that